test/utils/utils.test.js

```diff
@@ -572,6 +573,20 @@ describe('Utils', () => {
     });
   });
 
+  describe('createTag', async () => {
```

Comment:
@npeltier did `createTag` not have a UT prior to this diff?

Reply:
not specifically, but indirectly i think it was already 100% covered